app/src/main/java/ar/com/wolox/android/example/BaseConfiguration.kt

```diff
@@ -3,7 +3,7 @@ package ar.com.wolox.android.example
 internal open class BaseConfiguration {
 
     companion object {
-        const val EXAMPLE_CONFIGURATION_KEY = "https://jsonplaceholder.typicode.com"
+        const val EXAMPLE_CONFIGURATION_KEY = "https://w-android-training.herokuapp.com" // "https://jsonplaceholder.typicode.com"
```

Comment:
You can remove the `jsonplaceholder` string, we are not using it in this training